eqt/ui/UISliderWidget.py

```diff
@@ -6,18 +6,32 @@ class UISliderWidget(QSlider):
     '''Creates a Slider widget which updates
     a QLabel with its value (which may be scaled
     to a non-integer value by setting the scale_factor)'''
-    def __init__(self, label, scale_factor=1):
+    def __init__(self, dspinbox, min=0.00, max=1.00, scale_factor=1, parent=None):
         QSlider.__init__(self)
-        self.label = label
+        self.parent = parent
```

Comment:
please delete all the occurrences of self. parent. You can call the parent by using `parent()` in qt.

Reply:
References to `parent` attribute and parameter removed